test/integration/golden_response/staging/get_stat_ranking/country.json

```diff
@@ -20,1246 +20,1216 @@
           },
           {
             "rank": 4,
-            "value": 293047571,
-            "placeDcid": "country/SUN"
-          },
```

Comment:
This is expected because cl/359318531 drops dissolved countries from showing up in ranking.

Although submitted on Feb 24th, it was only rolled out on Mar 8th run, which would explain the diff showing up now.